tests/pytests/functional/formulas/test_docker.py

```diff
@@ -12,12 +12,17 @@ def _formula(saltstack_formula):
 
 @pytest.fixture(scope="module")
 def modules(loaders, _formula):
+    loaders.opts["file_roots"]["base"].append(
```

Comment:
@s0undt3ch is this okay to directly modify the opts on `loaders`? I checked when running two of these tests and it didn't seem to modify the opts for the next test, but wanted to confirm.

Reply:
Yes. The loaders are reset after each test, including opts.

https://github.com/saltstack/pytest-salt-factories/blob/master/src/saltfactories/utils/functional.py#L123